src/cockpit/channels/stream.py

```diff
@@ -72,6 +72,7 @@ def create_transport(self, loop: asyncio.AbstractEventLoop, options: Dict[str, A
         pty: bool = options.get('pty', False)
         window: Dict[str, int] = options.get('window')
         environ = options.get('environ')
+        directory = options.get('directory')
```

Comment:
This is already called `cwd`, 4 lines up . Possibly confusing.  Feel free to rename it.

Reply:
How did I read over that....